bot/bot.py

```diff
@@ -16,124 +17,132 @@
-
-	def help_message(self):
-		help_lines = []
-		help_lines.append("**Welcome to The Contract's Dice Rolling Bot!**")
-		help_lines.append("This bot can help you roll dice for the game The Contract https://www.TheContractRPG.com/ ")
-		help_lines.append("Contribute to my source code: {}".format(bot_repo_url))
-		help_lines.append("Add this bot to your server: {}".format(bot_invite_url))
-		help_lines.append(self.usage_message())
-		return "\n".join(help_lines)
-
-	def error_usage_message(self, error=None):
-		if error:
-			return "Error - {}\n{}".format(error, self.usage_message())
-		return self.usage_message()
-
-	def usage_message(self):
-		usage_lines = []
-		usage_lines.append("**Usage:**")
-		usage_lines.append("`{}{}` Display help".format(self.tokens["prefix"], self.tokens["help"]))
-		usage_lines.append("`{}3` Roll 3 dice difficulty 6".format(self.tokens["prefix"]))
-		usage_lines.append("`{}4{}8` Roll 4 dice difficulty 8".format(self.tokens["prefix"], self.tokens["roll_seperator"]))
-		usage_lines.append("`{}5 {}` Roll 5 dice difficulty 6 and exert".format(self.tokens["prefix"], self.tokens["exertion"]))
-		usage_lines.append("`{}3 init` Roll 3 dice difficulty 6 and label 'init'".format(self.tokens["prefix"]))
-		usage_lines.append("`{}4{}7 {} blue` Roll 4 dice difficulty 7, exert, and label 'blue'".format(self.tokens["prefix"], self.tokens["roll_seperator"], self.tokens["exertion"]))
-		usage_lines.append("`{}{}` Flip a coin calling 'high'".format(self.tokens["prefix"], self.tokens["flip_high"]))
-		usage_lines.append("`{}{}` Flip a coin calling 'low'".format(self.tokens["prefix"], self.tokens["flip_low"]))
-		return "\n".join(usage_lines)
-
-	def roll_dice_response(self, message, parsed_message):
-		first_word_tokens = parsed_message[0]
-		for token in first_word_tokens:
-			if not token.isdigit():
-				return self.error_usage_message("{} is not a number".format(token))
-		num_dice = int(first_word_tokens[0])
-		if num_dice >= 50:
-			return "Error - please roll fewer than 50 dice"
-		difficulty = int(first_word_tokens[1]) if len(first_word_tokens) > 1 else 6
-		exert = False
-		label = None
-		if len(parsed_message) > 1:
-			for token in parsed_message[1:]:
-				if token == self.tokens["exertion"]:
-					exert = True
-				else:
-					label = token
-		
-		
-		return self.contract_roll(num_dice=num_dice, difficulty=difficulty, exert=exert, label=label)
-
-	def contract_roll(self, num_dice, difficulty=6, exert=False, label=None):
-		gt_9_diff_text = ""
-		if difficulty > 9:
-			difference = difficulty - 9
-			num_dice -= max(difference, 0)
-			difficulty = 9
-			gt_9_diff_text = "(Difficulty > 9 decreases dice)"
-		results = [random.randint(1, 10) for x in range(num_dice)]
-		results.sort(reverse=True)
-		outcome = 0
-		for res in results:
-			if res >= difficulty:
-				outcome += 1
-			if res == 10:
-				outcome += 1
-			if res == 1:
-				outcome -= 1
-		if exert:
+def help_message():
+	help_lines = []
```

Comment:
This can just be the list's declaration instead of using append. 

```
help_lines = [
    "**Welcome to The Contract's Dice Rolling Bot!**",
    "This bot can help you roll dice for the game The Contract https://www.TheContractRPG.com/ ",
    "Contribute to my source code: {}".format(bot_repo_url),
]
```

etc. 

Note you could also use a block quote `""" text with newlines """` but I like using separate elements because it makes the formatting clearer.